src/client/src/components/Nav/Nav.tsx

```diff
@@ -6,15 +6,20 @@ import { connect } from 'react-redux'
 import { NavLink, Link } from 'react-router-dom'
 import { fetchProducts } from '../../store'
 
-const mapStateToProps = ({ user, order }: { user: User; order: Order[] }) => {
-  return { user, order }
+interface GHNavLink {
+  name: string
+  path: string
+}
+
+const mapStateToProps = ({ user, order, cartCount }: { user: User; order: Order[]; cartCount: number }) => {
+  return { user, order, cartCount }
 }
 
 const mapDispatchToProps = (dispatch: any) => ({
   fetchProducts: () => dispatch(fetchProducts())
 })
 
-const Nav = ({ user, order }: { user: User; order: Order[] }) => {
+const Nav = ({ user, order, cartCount }: { user: User; order: Order[]; cartCount: number }) => {
```

Comment:
if we're getting cartCount on props, do we still need the array of orders? I think the only point in getting the orders on props here was to have a number of items, which we accomplish with the cartCount.

Reply:
Good point. I'll refactor it accordingly

Reply:
Updated now